Services/ContainerReference/classes/class.ilContainerReferenceGUI.php

```diff
@@ -1,7 +1,5 @@
 <?php
 
-declare(strict_types=1);
```

Comment:
@nmatuschek We should not remove this, but only move this below the license header.